projects/form-demo-app/src/app/app.component.html

```diff
@@ -1,8 +1,11 @@
 
 <div style="margin: 25px;padding: 0px 30px;">
-  <h3>Form Demo</h3>
+  <!-- Charts implementation example -->
   <app-chart> </app-chart>
   <lib-table ></lib-table>
-  <dynamic-form [formJson]="data" #formLib></dynamic-form>
+  <!-- languageTranslate pipe implementation example-->
+  <div style="margin: 30px 0px;">{{ message | languageTranslate:language }}</div>
+  <!-- Dynamic form implementation example -->
+  <dynamic-form [formJson]="data" [language]="language" #formLib></dynamic-form>
   <button (click)="submitData()" [disabled]="this.formLib?.myForm?.invalid" style="margin: 30px 0px;">SUBMIT</button>
```

Comment:
Translate for submit

Reply:
This is just for demo app so this submit button won't show up in mitra or anywhere else we use